BootCamp.html

```diff
@@ -97,12 +97,12 @@
 	<h2>Getting Enzo</h2>
 
 	<p>The simplest way to get a copy of the current stable source code
-               is to clone the repository using Mercurial:</p>
-               <pre class="prettyprint">$ hg clone https://bitbucket.org/enzo/enzo-dev <br>$ hg update stable </pre>
+               is to clone the repository using git:</p>
+               <pre class="prettyprint">$ git clone https://github.com/enzo-project/enzo-dev </pre>
         </p>
 	
-	<p>Mercurial (hg) is a revision-control system that is available on many
-	platforms (see <a href="https://www.mercurial-scm.org">www.mercurial-scm.org</a>).  (Alternately if you prefer you can update to "week-of-code" instead of "stable" to use the development version).
+	<p>Git is a revision-control system that is available on many
```

Comment:
Might as well change this while we're at it.
```suggestion
	<p>Git is a version control system that is available on many
```